recipes/sdl/all/conanfile.py

```diff
@@ -0,0 +1,318 @@
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+
+    def _add_libraries_from_pc(self, library, static=None):
+        if static is None:
+            static = not self.options.shared
+        pkg_config = tools.PkgConfig(library, static=static)
+        libs = [lib[2:] for lib in pkg_config.libs_only_l]  # cut -l prefix
+        lib_paths = [lib[2:] for lib in pkg_config.libs_only_L]  # cut -L prefix
+        self.cpp_info.components["libsdl2"].libs.extend(libs)
+        self.cpp_info.components["libsdl2"].libdirs.extend(lib_paths)
+        self.cpp_info.components["libsdl2"].sharedlinkflags.extend(pkg_config.libs_only_other)
+        self.cpp_info.components["libsdl2"].exelinkflags.extend(pkg_config.libs_only_other)
+
+    def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "SDL2"
+        self.cpp_info.names["cmake_find_package_multi"] = "SDL2"
+
+        postfix = "d" if self.settings.build_type == "Debug" else ""
+        # SDL2
+        sdl2_cmake_target = "SDL2" if self.options.shared else "SDL2-static"
+        self.cpp_info.components["libsdl2"].names["cmake_find_package"] = sdl2_cmake_target
+        self.cpp_info.components["libsdl2"].names["cmake_find_package_multi"] = sdl2_cmake_target
+        self.cpp_info.components["libsdl2"].includedirs.append(os.path.join("include", "SDL2"))
+        self.cpp_info.components["libsdl2"].libs = ["SDL2" + postfix]
+        if self.options.get_safe("iconv", False):
+            self.cpp_info.components["libsdl2"].requires.append("libiconv::libiconv")
+        if self.settings.os == "Linux":
+            self.cpp_info.components["libsdl2"].system_libs = ["dl", "rt", "pthread"]
+            self.cpp_info.components["libsdl2"].requires.append("xorg::xorg")
+            if self.options.alsa:
+                self.cpp_info.components["libsdl2"].requires.append("libalsa::libalsa")
+            if self.options.pulse:
+                self.cpp_info.components["libsdl2"].requires.append("pulseaudio::pulseaudio")
+            if self.options.opengl:
+                self.cpp_info.components["libsdl2"].requires.append("opengl::opengl")
+            if self.options.jack:
+                self._add_libraries_from_pc("jack")
+            if self.options.sndio:
+                self._add_libraries_from_pc("sndio")
+            if self.options.nas:
+                self.cpp_info.components["libsdl2"].libs.append("audio")
+            if self.options.esd:
+                self._add_libraries_from_pc("esound")
+            if self.options.directfb:
+                self._add_libraries_from_pc("directfb")
+            if self.options.video_rpi:
+                self.cpp_info.components["libsdl2"].libs.append("bcm_host")
+                self.cpp_info.components["libsdl2"].includedirs.extend([
+                    "/opt/vc/include",
+                    "/opt/vc/include/interface/vcos/pthreads",
+                    "/opt/vc/include/interface/vmcs_host/linux"
```

Comment:
```suggestion
                    "/opt/vc/include",
                    "/opt/vc/include/interface/vcos/pthreads",
                    "/opt/vc/include/interface/vmcs_host/linux"
```
Why those absolute path, can it be different? You add options and remove them from package ID, so the user can pass any path.

Reply:
they are hard-coded in RPI AFAIK